l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -256,7 +256,7 @@ def getCedPrest(self, cedPrest):
         if partner_id and not no_contact_update:
             partner_company_id = partner_model.browse(partner_id).company_id.id
             vals = {
-                'street': cedPrest.Sede.Indirizzo,
+                'street': str(f"{cedPrest.Sede.Indirizzo} {str(cedPrest.Sede.NumeroCivico)}").strip(),
```

Comment:
Guardando gli errori Travis, andrebbe aggiunto solo se diverso da None, altrimenti salta fuori questo:

```2021-07-29 10:19:23,780 8307 ERROR openerp_test odoo.addons.l10n_it_fatturapa_in.tests.test_import_fatturapa_xml: ` AssertionError: 'VIALE ROMA 543 None' != 'VIALE ROMA 543'```

Reply:
Per risolvere entrambe le problematiche (versione python* + numero civico assente) si potrebbe procedere in questo modo?

if partner_id and not no_contact_update:
            partner_company_id = partner_model.browse(partner_id).company_id.id
            **street = cedPrest.Sede.Indirizzo
            if cedPrest.Sede.NumeroCivico:
                street += " " + str(cedPrest.Sede.NumeroCivico)**
            vals = {
                **'street': street,**
                'zip': cedPrest.Sede.CAP,
                'city': cedPrest.Sede.Comune,
                'register': cedPrest.DatiAnagrafici.AlboProfessionale or ''
            }

Reply:
O anche:
```
if cedPrest.Sede.NumeroCivico:
    street += " " + str(cedPrest.Sede.NumeroCivico)
```
o se ti senti pythonico, e vuoi farlo su una riga, qualcosa del genere: :)
```
vals = {
                "street": " ".join(map(str, filter(None, (cedPrest.Sede.Indirizzo, cedPrest.Sede.NumeroCivico))))
```

Reply:
Beh, non mi lascio sfuggire l'occasione di essere più pythonico :-)) 
Ho aggiornato la PR
Grazie @TheMule71 !